src/server/common/model/page/page-model.js

```diff
@@ -7,4 +10,40 @@ export class Page {
 
   /** @type {string} */
   sectionKey
+
+  /** @type {string} */
+  key
+
+  /** @type {string} */
+  view
+
+  /** @type {string} */
+  pageHeading
+
+  /** @type {string} */
+  pageTitle
+
+  /** @returns {string} */
+  get heading() {
+    return this.pageHeading
+  }
+
+  /** @returns {string} */
+  get title() {
+    return this.pageTitle
+  }
+
+  /** @returns {Record<string, unknown>} */
+  get viewProps() {
+    return {}
+  }
```

Comment:
I think I like this!

Reply:
so do i which is why i left it - it was attempt #1 of getting the continue button to look like a link, but there is no GDS style i could find for it, but liked it so left it